eval/builtin_ns.go

```diff
@@ -13,6 +14,7 @@ var builtinNs = Ns{
 	"ok":    vars.NewRo(OK),
 	"true":  vars.NewRo(true),
 	"false": vars.NewRo(false),
+	"nil":   vars.NewRo(vals.Nil{}),
```

Comment:
I don't want to introduce a nil value for Elvish. Call this `-json-nil`.

Reply:
Can do.

Reply:
Sorry for the late feedback, but now that this is only used for JSON, `json-null` is actually a more sensible choice...